### PR TITLE
Documentation updates

### LINKED ISSUE #1595: syliusBundle sm.factory
Hi, 

since last commit, there is an error on an undefined service: 

[Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException]
  The service "sylius.processor.shipment_processor" has a dependency on a non-existent service "sm.factory".

Cheerz

### PR DESCRIPTION
**Pull request by kshishkin**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets | fixes #1595, partially #3750 
| License         | MIT

### CODE REVIEW

File: docs/bundles/SyliusSettingsBundle/installation.rst [hunk 1]
```diff
@@ -33,6 +33,7 @@ This bundle also uses `DoctrineCacheBundle`. Don't worry, everything was automat
     public function registerBundles()
     {
         $bundles = array(
+            new winzou\Bundle\StateMachineBundle\winzouStateMachineBundle(),
```
Comment (bretto36) [inline docs/bundles/SyliusSettingsBundle/installation.rst]: There is no leading slash on this one like there is on the others

**Comment by kshishkin** [comment]
Unified, apology.

**Comment by lchrusciel** [comment]
But, can you squash these commits? 13 commits to change 10 lines it is a little bit too much.

**Comment by TheMadeleine** [comment]
Looks like you've got multiplicated commits, they even have the same messages.

**Comment by kshishkin** [comment]
You're right. I'll sqash once I'm at my desk. Pardon the hassle, I'm yet to find my ways around git & github. Multiple commits came from online editing.

-----Wiadomość oryginalna-----
Od: "Magdalena Banasiak" <notifications@github.com>
Wysłano: ‎2016-‎09-‎03 20:49
Do: "Sylius/Sylius" <Sylius@noreply.github.com>
DW: "Krzysztof Lechowski" <krzysztof.lechowski@gmail.com>; "Author" <author@noreply.github.com>
Temat: Re: [Sylius/Sylius] Documentation updates (#5966)

Looks like you've got multiplicated commits, they even have the same messages.

**Pull request closed without merging**